etl/pipelines/arkg_builder_pipeline.py

```diff
@@ -1,91 +1,106 @@
-                Literal(record_key),
-            )
-        )
+        `item_reviewed` is the IRI of the item that is being anti-recommended.
 
-    def __add_url_quad_to_store(self, record_key: RecordKey) -> None:
+        Each anti_recommendation_key has 2 Quad expressions in this method:
+        - a `type` Quad that expresses the type of Review the anti_recommendation_key belongs to.
+        - an `item-reviewed` Quad that relates the anti_recommendation_key to item that is being anti-recommended.
         """
-        Add a RDF Quad with a `HAS_URL` predicate to the RDF Store.
+        for anti_recommendation_key in anti_recommendation_keys:
 
-        The subject of the Quad is the Store's base iri + record_key.
-        The object of the Quad is Wikipedia's base url + record_key.
-        """
+            anti_recommendation_instance = ArkgInstance.anti_recommendation_iri(
+                record_key=anti_recommendation_key.replace(" ", "_")
+            )
 
-        self.__store.add(
-            Quad(
-                NamedNode(self.__base_iri + record_key.replace(" ", "_")),
-                NamedNode(self.__base_iri + Predicate.HAS_URL.value),
-                Literal(WIKIPEDIA_BASE_URL + record_key),
+            self.__store.add(
+                Quad(
+                    anti_recommendation_instance,
+                    RDF_TYPE,
+                    ArkgSchema.RECOMMENDATION,
+                )
+            )
+            self.__store.add(
+                Quad(
+                    anti_recommendation_instance,
+                    ArkgSchema.ITEMREVIEWED,
+                    NamedNode(item_reviewed),
+                )
             )
-        )
 
-    def __add_anti_recommendation_quads_to_store(
+    def construct_graph(
         self,
-        anti_recommendation_graph: tuple[RecordKey, tuple[AntiRecommendationKey, ...]],
-    ) -> None:
+        graphs: tuple[tuple[RecordKey, tuple[AntiRecommendationKey, ...]], ...],
+    ) -> Store:
         """
-        Add RDF Quads with `HAS_ANTI_RECOMMENDATION` predicates to the RDF Store.
+        Return a RDF Store populated with anti_recommendation_graphs.
+
+        Each record_key in the anti_recommendation_graph has 3 Quad expressions in this method:
+        - a `type` Quad that expresses the type of CreativeWork the record_key belongs to.
+        - a `title` Quad that expresses the title of the record_key.
+        - a `url` Quad that expresses the URL of the record_key.
 
-        The subject of each Quad is the Store's base iri + record_key.
-        The object of each Quad is the Store's base iri + anti_recommendation_key.
+        All anti-recommendations are related to a record_key via the __add_anti_recommendation_quads_to_store method.
         """
 
-        for anti_recommendation_key in anti_recommendation_graph[1]:
+        for graph in graphs:
+            record_key = graph[0].replace(" ", "_")
```

Comment:
Actually, you could probably put a `to_iri` and `from_iri` on RecordKey instead of in the IRI factory method. Either way, it shouldn't be spread out.